app/scripts/controllers/swaps.js

```diff
@@ -625,36 +672,44 @@ export default class SwapsController {
 }
 
 /**
- * Calculates the median of a sample of BigNumber values.
+ * Calculates the median ethValueOfQuote of a sample of quotes.
  *
- * @param {import('bignumber.js').BigNumber[]} values - A sample of BigNumber
- * values. The array will be sorted in place.
- * @returns {import('bignumber.js').BigNumber} The median of the sample.
+ * @param {array} quotes - A sample of quote objects with ethValueOfQuote and ethFee properties
+ * @returns {object} An object with the media ethValueOfQuote for all quotes, and the associated median ethFee and metaMaskFeeInEth.
  */
-function getMedian (values) {
-  if (!Array.isArray(values) || values.length === 0) {
+function getMedianEthValueQuote (quotes) {
+  if (!Array.isArray(quotes) || quotes.length === 0) {
     throw new Error('Expected non-empty array param.')
   }
 
-  values.sort((a, b) => {
-    if (a.equals(b)) {
+  quotes.sort(({ ethValueOfQuote: ethValueOfQuoteA }, { ethValueOfQuote: ethValueOfQuoteB }) => {
+    if (ethValueOfQuoteA.equals(ethValueOfQuoteB)) {
       return 0
     }
-    return a.lessThan(b) ? -1 : 1
+    return ethValueOfQuoteA.lessThan(ethValueOfQuoteB) ? -1 : 1
```

Comment:
Shouldn't this be `isLessThan`? What BigNumber library is being used here?

Reply:
In the [changelog for BigNumber version 6.0](https://github.com/MikeMcl/bignumber.js/blob/master/CHANGELOG.md#600), there is `Rename methods: add is prefix to greaterThan, greaterThanOrEqualTo, lessThan and lessThanOrEqualTo`

In our package.json is `"bignumber.js": "^4.1.0"`

Reply:
Ah I see! I actually did check the correct version of that library, but the image in the README is updated to reflect the latest version :facepalm: